doc/api/tools-bilateralize.rst

```diff
@@ -0,0 +1,97 @@
+.. currentmodule:: mesage_ix_models.tools.bilateralize
+Bilateralize (:mod:`.tools.bilateralize`)
+#########################################
+
+.. contents::
+  :local:
+  
+Overview
+========
+This documentation will outline how to bilateralize trade flows in MESSAGEix. By default, trade flows are based on a “global pool” approach, wherein all exporters export energy/commodities to a pool from which importers import based on regionally resolved demands.
+
+By “bilateralizing” trade, we specify exporter and importer flows and can therefore represent trade flows as networks.
+
+The bilateralization tool, **bilateralize**, is generalized for any traded commodity, whether that is a fuel (e.g., LNG), or a material (e.g., steel). It also explicitly represents bilateral trade “flows”, or how a fuel/commodity is transported from exporter to importer. These flow technologies are user defined and flexible; the most common are pipelines (e.g., gas pipelines), maritime shipping (e.g., LNG tanker), and transmission lines.
+
+The ``bilateralize`` tool is a Python script that can be used to bilateralize trade flows in MESSAGEix. It is located in the ``message-ix-models/tools/bilateralize`` directory.
+
+The tool follows the following steps, which are also available in ``tool/bilateralize/workflow.py``:
+
+Step 1 | Edit (``tools/bilateralize/prepare_edit.py``)
+=============
+The first step is to generate empty (or default valued) parameters that are required for bilateralization, specified by commodity. This step requires updates to a configuration file (``config.yaml``) that should be housed in a project directory (e.g., ``message-ix-models/projects/newpathways-trade/config.yaml``). A template configuration file is provided at ``message-ix-models/data/bilateralize/configs/base_config.yaml``. 
+
+Once the configuration is updated, the user can run ``message_ix_models.tools.bilateralize.prepare_edit.generate_edit_files(log, project_name, config_name, message_regions)`` to produce empty (or default valued) parameters as CSV files. These CSV files will populate in ``message-ix-models/data/[your_trade_commodity]/edit_files``. 
```

Comment:
Please see the existing documentation and the Sphinx docs ([1](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html), [2](https://www.sphinx-doc.org/en/master/usage/referencing.html), [3](https://www.sphinx-doc.org/en/master/usage/domains/python.html#cross-referencing-python-objects)) for how to use 'roles'.

Some that could be used in this file include:
```rest
:file:`message_ix_models/data/bilateralize/configs/base_config.yaml`
:func:`.bilateralize.prepare_edit.generate_edit_files`
:mod:`.bilateralize.workflow`
```